src/entities/catalogs/services/get-pageviews-by-catalog-id.ts

```
@@ -0,0 +1,76 @@
+) {
+  // Check if response exists and has rows
+  if (!response?.rows) {
+    return [];
+  }
+
+  // Map through the rows and create an object with id and value of the catalog
+  return response.rows.map((row) => {
+    if (row.dimensionValues && row.metricValues) {
+      return {
+        id: row?.dimensionValues[0]?.value,
+        pageviews: parseInt(row?.metricValues[0]?.value ?? "0", 10),
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)